master/buildbot/process/build.py

```diff
@@ -533,22 +538,21 @@ def buildFinished(self, text, results):
         state.
 
         It takes two arguments which describe the overall build status:
-        text, results. 'results' is one of SUCCESS, WARNINGS, or FAILURE.
+        text, results. 'results' is one of the possible results (see buildbot.status.results).
```

Comment:
I actually don't know which doc generation tool is used in buildbot, (and thus
the @type @param are more aimed at being understood by the programmer reading the code rather than a programmer reading the doc). What do you think @tardyp  @djmitche @sa2ajj ?